CONTRIBUTING.md

```diff
@@ -0,0 +1,179 @@
+
+The browser-based workflow might be sufficient for smaller improvements, but for larger contributions,
+especially if you want to add new pages or intend to contribute for multiple times,
+the recommended way is cloning the forked project to your machine and preparing your changes locally in a suited text editor.
+This makes it easier to write clean files and to preview the result in the local development server before any PR is submitted.
+
+### Workflow overview
+
+This actually applies to the browser-based workflow, except that some steps in the browser are shortened a bit.
+And of course both can be combined, e.g., one can open a PR in the browser but finish it locally.
+
+1. Setup
+   - Make sure you have [git] installed
+   - If you want to preview the site in a local development server, install [hugo].
+   - If you want to edit R Markdown files, you'll also need [R] and the [blogdown] package.
+   - If you don't have write access to the project repo, fork it.
+   - Clone the repo you want to write to
+   - Create a new local branch from the main branch, name it after your contribution
+2. Uploading your Changes (iterative)
+   - Make your improvements
+   - Please try to write one sentence or clause per line, as far as it makes sense. This improves git diff and file reviews.
+   - Try to format your edited files with [Prettier] and lint Markdown with [markdownlint].
+   - commit to the new branch.
+   - Push the new branch to your remote
+3. Requesting the improvements to be merged
+   - On Github, create a pull request to merge the new branch into the main branch
+   - If the title is not sufficient, you can provide a PR description.
+   - Please leave the “allow edits” checkbox enabled
+4. Checking, cleaning (iterative)
+   - When a PR is created, Prettier runs on the PR branch.
+     If it detects unformatted files, a commit that applies formatting changes is added to the branch.
+     if you want to add more commits to the PR, you will have to pull these changes into your local branch.
+   - After prettifying is done, the Markdown files are checked for issues that relate to ambiguity, accessibility, consistency, and alike by [markdownlint].
+     If this check doesn't pass, you have to add commits to your branch which fix the detected issues.
+   - The checks rerun and the cycle repeats, until the checks succeed.
+5. Reviewing and improving content (iterative)
+   - Now your PR is ready for review, and you can request review from one or more collaborators.
+   - If a reviewer rejects your PR and provides feedback, you will have to discuss or integrate the feedback
+     by adding fixup commits to your PR branch until it gets approved.
+     This also triggers the checking and cleaning cycle.
+   - Meanwhile, if the main branch receives new commits and your PR can't be merged anymore due to merge conflicts,
+     please don't merge main into your branch, but rebase your branch on main and force-push your branch.
```

Comment:
this needs explanation or a link.

Reply:
Added this link: <https://www.atlassian.com/git/tutorials/merging-vs-rebasing>